internal/cli/ops_manager_alerts_global_list.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	om "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type opsManagerAlertsGlobalListOpts struct {
+	store        store.GlobalAlertLister
+	pageNum      int
+	itemsPerPage int
+	status       string
+}
+
+func (opts *opsManagerAlertsGlobalListOpts) init() error {
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *opsManagerAlertsGlobalListOpts) Run() error {
+	alertOpts := &om.AlertsListOptions{
+		Status: opts.status,
+		ListOptions: mongodbatlas.ListOptions{
```

Comment:
just as FYI (no need for any change here) you can also do
```go
alertOpts := &om.AlertsListOptions{}
alertOpts.PageNum = opts.pageNum
```
go has automatic delegation for embedded structs, this is just style and there's no advantage of one over the other, so just letting you know

Reply:
Also for future work, we should do with pagination what we did with delete that we embed a struct with the common things that all paginated resources do

Reply:
I will create the activity on Jira